docs/sdk/core-kit/mpc-core-kit/usage.mdx

```diff
@@ -51,6 +54,41 @@ Additional Helper Functions:
   - `mnemonicToKey()` - Converts a Mnemonic to a BN.
   - `keyToMnemonic()` - Converts a hex string to a Mnemonic.
 
+## Set TSS Wallet Index
+
+#### `setTssWalletIndex(accountIndex: number): void;`
+
+Sets the TSS Wallet Index for the User's account. This is a new feature that supports key derivation, allowing users to derive multiple private keys
+from a single account for improved asset organization.
+
+#### Usage
+
+```tsx
+await coreKitInstance.setTssWalletIndex(index);
+```
+
+## Get Public Sync
+
+#### `getPublicSync()`
+
+### Parameters
+
+```js
+public getPublicSync: () => Buffer = () => {
+  let { tssPubKey } = this.state;
+  if (tssPubKey.length === FIELD_ELEMENT_HEX_LEN + 1) {
+    tssPubKey = tssPubKey.subarray(1);
+  }
+  return Buffer.from(tssPubKey);
+};
+```
+
+#### Usage
+
+```ts
+await coreKitInstance.getPublicSync();
```

Comment:
@ieow is this function returns final public key of the user?

Reply:
also the public key will depends on account index being set, so for different account indexes this function will return different public keys. We should mention it in docs.

Reply:
Updated!